images/bootstrap/Dockerfile

```diff
@@ -106,7 +106,7 @@ RUN curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID"
 # Trying to remount these makes for a very noisy error block in the beginning of
 # the pod logs, so we just comment out the call to it... :shrug:
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends docker-ce=5:20.10.* && \
```

Comment:
Docker has been 20.10.X since 2020. This wasn't doing anything and docker has been plenty stable.